File: qualifier-week-4/graded-assignments/GrPA5 - Graded.c
#include <stdio.h>

int series_sum(int n) {
    int sum = 0;
    for (int i = 1; i <= n; i++) {
        int term_sum = 0;
        for (int j = 1; j <= i; j++) {
            term_sum += j;
        }
        sum += term_sum;
    }
    return sum;
}

int main() {
    int n;

    scanf("%d", &n);

    int sum = series_sum(n);
    printf("%d\n", sum);

    return 0;
}

File: qualifier-week-4/graded-assignments/GrPA5 - Graded.rs
use std::io;

fn series_sum(n: i32) -> i32 {
    let mut sum = 0;
    for i in 1..=n {
        sum += (1..=i).sum::<i32>();
    }
    sum
}

fn main() {
    let mut input = String::new();

    // Input the number of terms
    println!("Enter the number of terms: ");
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let n: i32 = input.trim().parse().expect("Invalid input");

    // Calculate the sum of the series
    let sum = series_sum(n);

    // Print the result
    println!("{}", sum);
}
